op_rewards_tracking/readme.md

```diff
@@ -1,8 +1,28 @@
 ## Data Sources
-- [OP Incentive Program Info](https://www.notion.so/oplabs/26d856d5ad7c4fda919c62e839cf6051?v=4e38174b2e994129b51dcfa127965aa4)
+- [OP Incentive Program Info](https://www.notion.so/oplabs/26d856d5ad7c4fda919c62e839cf6051?v=4e38174b2e994129b51dcfa127965aa4) (requires manual update)
```

Comment:
Update to the public Notion link: https://oplabs.notion.site/26d856d5ad7c4fda919c62e839cf6051?v=4e38174b2e994129b51dcfa127965aa4

From Share > Share to Web in Notion